src/styles/index.css

```diff
@@ -0,0 +1,18 @@
+/* Error: Undefined variable.
+ *   ,
+ * 4 |   gap: $gap;
```

Comment:
The error message indicates that the variable `$gap` is undefined. Make sure that the variable is defined in a file that is imported before it is used in `calendar.scss`.